### PR TITLE
Ultra disk

### PR DESCRIPTION
**Pull request by anasouma**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 201-ultra-managed-disk/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            512,
+            4096
+        ],        
+        "metadata": {
+          "description": "Logical Sector Size. Recommended 4096 unless application requires 512 bytes sector size support"
+        }
+      },
+      "diskSizeinGiB": {
+        "type": "int",
+        "defaultValue": 8,
```
Comment (bmoore-msft) [inline 201-ultra-managed-disk/azuredeploy.json:24]: are there min/max or allowedValues on these next few params?

File: 201-ultra-managed-disk/azuredeploy.parameters.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```
Comment (bmoore-msft) [inline 201-ultra-managed-disk/azuredeploy.parameters.json:4]: remove this and use defaultValues in the template file

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 201-ultra-managed-disk/azuredeploy.json (hunk 1 above)
Comment (anasouma) [reply 201-ultra-managed-disk/azuredeploy.json:24]: Address this in latest change

File: 201-ultra-managed-disk/azuredeploy.parameters.json (hunk 2 above)
Comment (anasouma) [reply 201-ultra-managed-disk/azuredeploy.parameters.json:4]: Addressed some of the variables. Please let me know if there are others you are concerned about.

**Pull request merged**